### DISCUSSION TITLE
No Leveling Tracker?

### DISCUSSION THREAD

**Discussion by JacksonnnT** [Lailloken/Exile-UI#434] [Q&A] [answered]
Hello so I came across this tool and wanted to have it setup and installed the programs including autohotkey 1.137 latest before 2.0 version I shouldn't use and even tried 1.136.02 but no matter what after opening the tool ingame I don't see the leveling tracker can someone help me thanks in advance

**Comment by Lailloken** [comment, marked as answer]
I recently renamed it to ` act-tracker ` in order to emphasize that it's a campaign-related feature.

**Reply by JacksonnnT** [reply]
oh wow I see that now thank you for your help much appreciated
